pipework

```diff
@@ -190,6 +190,13 @@ else
   }
 fi
 
+# This avoids segfault when pipework is called twice on the same container
+[ -L /sys/class/net/${IFNAME}/upper_ph${NSPID}${CONTAINER_IFNAME} ] &&
+  echo "A bridge on ${CONTAINER_IFNAME} already exists for ${GUESTNAME}" &&
+  IFS=$(ls /sys/class/net/${IFNAME}/upper_ph${NSPID}* | sed "s/\/sys\/class\/net\/${IFNAME}\/upper_ph${NSPID}//g") &&
```

Comment:
OK, so I think we shouldn't use `IFS`, since it's a built-in variable :-)

I'm not sure I understand the goal of that line... Can you explain what it does? It should list the content of a directory, but I don't understand why, sorry!